include/nbl/builtin/hlsl/subgroup/fft.hlsl

```diff
@@ -11,6 +11,8 @@ namespace hlsl
 {
 namespace subgroup
 {
+namespace fft
+{
```

Comment:
I did not mean for you to add an `fft` namespace to the `FFT` struct, just the `exchange` function!